nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
 
+  // Reset profiling timers and counters within event_delivery_manager
+  kernel().event_delivery_manager.reset_timers_counters();
```

Comment:
✅ 
Should be or maybe? Should they be reset every time you call run, or only once for a simulate function? Wouldn't this change the meaning, where before you called simulate (and the counters where reset) and then resume multiple times?